action_server/src/action_server/actions/count_and_tell.py

```diff
@@ -8,11 +8,13 @@
 
 
 class CountAndTell(Action):
-    ''' The CountAndTell class implements the action to count the number of objects and tell the operator.
+    """
+    The CountAndTell class implements the action to count the number of objects in a specific location and tell the
+        operator the amount of objects, the type and the location.
 
     Parameters to pass to the configure() method are:
-     - `sentence` (required): The sentence to speak. May be a keyword to tell something more intelligent.
-    '''
+     - `config` (required): the ConfigurationData defines the input data structure for configuration of an action.
```

Comment:
Is this doc relvant to this class? Seems very different from the original? 

Also, maybe it should be the docstring oth the `configure` method itself